### DISCUSSION TITLE
Is it possible to install Rehike on Windows XP?

### DISCUSSION THREAD

**Discussion by Chintrola** [Rehike/Rehike#356] [Q&A] [answered]
c

**Comment by ghost** [comment, marked as answer]
If you can get PHP 8 working on Windows XP (as well as a compatible web server), then yes. I am not sure to what effect that is possible, and I have never tried before.

We currently do not have plans to officially support Windows XP, because it is a 22 year old operating system with very outdated tools available for it, and our current framework is incompatible with it. Although transpilation to older PHP versions may be considered as a future option, even if it were to work well, there would likely be severe performance penalties and bugs due to design flaws and lack of optimizations in old versions of PHP.

**Comment by possiblynugg3t** [comment]
i was able to get it working by running it on another pc on the same network, and installing the rehike browser certificates and proxying via the hosts file on the xp machine, but instead of 127.0.0.1 for the hosts file, i put the local ip of the machine running it, also i needed to disable ipv6 for it to work for some reason

**Reply by ghost** [reply]
But this is not installing it on Windows XP. It's installing it on another computer and just using it on Windows XP.
